third_party/BUILD

```diff
@@ -563,12 +563,13 @@ filegroup(
         "//src/conditions:darwin": ["turbine/turbine_direct.darwin"],
         "//src/conditions:darwin_x86_64": ["turbine/turbine_direct.darwin"],
         "//src/conditions:linux_x86_64": ["turbine/turbine_direct.linux"],
-        "//conditions:default": [],
+        "//conditions:default": ["turbine/turbine_direct.jar"],
```

Comment:
This is more than renaming, can you mention it in the PR description?

Reply:
Done